lib/tower_cli/utils/types.py

```diff
@@ -65,7 +67,7 @@ class Related(click.types.ParamType):
     """A subclass of click.types.ParamType that represents a value
     related to another resource.
     """
-    name = 'related'
```

Comment:
I think that removing `name` is likely to cause problems because other things will assume that it exists.